### PR TITLE
Add channel param to options

### PR DESCRIPTION
**Pull request by jncorpron**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,7 +50,8 @@ Returns a [Promise](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Refe
 
   Key | Description | Default
   :--|:--|:--
-  `client` | [Client ID](https://developers.google.com/maps/documentation/javascript/get-api-key#specify-a-client-id-when-loading-the-api) | `undefined`
+  `channel` | [Client usage reporting channel](https://developers.google.com/maps/documentation/javascript/get-api-key#clientID-features) | `undefined`
+  `client` | [Client ID](https://developers.google.com/maps/documentation/javascript/get-api-key#clientID-features) | `undefined`
```
Comment (yuanqing) [inline README.md]: This line shouldn't have been changed

File: README.md [hunk 2]
```diff
@@ -50,7 +50,8 @@ Returns a [Promise](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Refe
 
   Key | Description | Default
   :--|:--|:--
-  `client` | [Client ID](https://developers.google.com/maps/documentation/javascript/get-api-key#specify-a-client-id-when-loading-the-api) | `undefined`
+  `channel` | [Client usage reporting channel](https://developers.google.com/maps/documentation/javascript/get-api-key#clientID-features) | `undefined`
```
Comment (yuanqing) [inline README.md]: Consider changing the link to https://developers.google.com/maps/premium/reports/usage-reports#channels

**Review by yuanqing — CHANGES_REQUESTED** [review]
@jncorpron Thanks for this. Happy to merge when the requested changes are made

File: README.md (hunk 1 above)
Comment (jncorpron) [reply README.md]: It seems that the anchor is no longer in the documentation, so I updated it to point to that new anchor. It seems that there is another anchor, similar to the original: https://developers.google.com/maps/documentation/javascript/get-api-key#specifying-a-client-id-when-loading-the-api

**Comment by jncorpron** [comment]
The link for the `key` option also has a broken anchor. It could be updated to be: https://developers.google.com/maps/documentation/javascript/get-api-key#step-2-add-the-api-key-to-your-application

**Comment by yuanqing** [comment]
Got it. Thanks @jncorpron! 🙌

**Review by yuanqing — APPROVED** [review]

**Pull request merged**